### PR TITLE
Include BrainKB UI origin in CORS middleware config for queryservice

### PR DESCRIPTION
**Pull request by aaronkanzer**
@tekrajchhetri this is the origination of your current CORS error seen in https://github.com/sensein/brainkb-ui/issues/6#issuecomment-2429432000

Relates to https://github.com/sensein/brainkb-ui/issues/6

Cc @kabilar

### CODE REVIEW

File: query_service/core/main.py [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "http://127.0.0.1:3001",
     "http://3.134.90.242", #aws ip
     "http://18.119.65.244", #graphdb
```
Comment (aaronkanzer) [inline query_service/core/main.py:24]: @kabilar @tekrajchhetri -- **note** -- unless you have allocated an AWS Elastic IP address to these IPs, the IPs will change each time if the instance goes up/down/rebooted, etc.

File: query_service/core/main.py [hunk 2]
```diff
@@ -22,6 +22,7 @@
...
+    "https://beta.brainkb.org",
```
Comment (aaronkanzer) [inline query_service/core/main.py:25]: @tekrajchhetri I'll leave it up to you if you want to also add any other origins

File: query_service/core/main.py (hunk 1 above)
Comment (tekrajchhetri) [reply query_service/core/main.py:24]: @aaronkanzer Thank you for the suggestion. I would work on your suggestion and put it on the active targets.

File: query_service/core/main.py (hunk 2 above)
Comment (tekrajchhetri) [reply query_service/core/main.py:25]: @aaronkanzer thank you. Could you update the pull request to merge to `[rapid-release-api-support](https://github.com/sensein/BrainKB/tree/rapid-release-api-support)`. It's the latest update I am working on atm and is not yet merged to main.

Comment (tekrajchhetri) [reply query_service/core/main.py:25]: merging --> ignore my previous comment.

File: query_service/core/main.py (hunk 1 above)
Comment (kabilar) [reply query_service/core/main.py:24]: Thanks, Aaron.  That's a good catch.

**Pull request merged**